src/include/sof/trace/trace.h

```diff
@@ -306,7 +306,7 @@ struct tr_ctx {
  * @param default_log_level Default log level.
```

Comment:
Cc: @crazoes 

@ktrzcinx looks good to me. Can you add a little bit more information in the commit message or here in comments on why do we want to iterate over the trace context entries. 

And what does it mean 'the right one'? What are we looking for. Maybe describe a little bit how this works.

Reply:
I extended commit description

Reply:
@ktrzcinx thanks! Looks good to me.